lib/hanami/hanamirc.rb

```diff
@@ -140,7 +140,12 @@ def symbolize(hash)
       Utils::Hash.new(hash).symbolize!
     end
 
+    # Checks file existence and return parsed values or empty hash
```

Comment:
`return` -> `returns`

Reply:
WDYT about this: `Returns options from hanamirc file`?

Reply:
@davydovanton thank you for review! Yea I think it better `Returns options from hanamirc file`